1209/EAEA/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: pid
+title: Pinscape Controller
```

Comment:
Thanks for catching that.  I just made the change you recommended.
Date: Tue, 29 Sep 2015 13:24:37 -0700
From: notifications@github.com
To: pidcodes.github.com@noreply.github.com
CC: mjr_@hotmail.com
Subject: Re: [pidcodes.github.com] Added Pinscape controller as 1209/EAEA (#113)

In 1209/EAEA/index.md:

> @@ -0,0 +1,13 @@
> +---
> +layout: pid
> +title: Pinscape Controller

FYI, as-is this will cause it to show up in the list as "Pinscape Pinscape Controller". Delete "Pinscape" here if you don't want that to be the case, or let me know if you want to go ahead and merge it as-is.


—
Reply to this email directly or view it on GitHub.